app/models/audit.rb

```diff
@@ -7,6 +7,13 @@ class Audit < ApplicationRecord
 
   scope :by_created_at, -> { order(created_at: :asc) }
   scope :with_user_and_api_user, -> { preload(:user, :api_user) }
+  scope :with_planning_application, -> { includes(:planning_application) }
+  scope :not_by_assigned_officer, lambda {
+    joins(:planning_application).where("audits.user_id != planning_applications.user_id OR planning_applications.user_id IS NULL")
+  }
+  scope :most_recent_for_planning_applications, lambda {
```

Comment:
This should just create the one query.

Essentially we want to grab the audit with the max created at date (i.e. most recent) per planning application and then group these by the id so we don't duplicate planning applications in the scope. We also only care about audits created by someone other than the assigned officer so we require the scope `not_by_assigned_officer`